businessRules/donations.js

```diff
@@ -21,34 +21,26 @@ function createDonation(firstName, sum, userId, ideaId) {
 }
 
 function ideasDonationSum(ideaId) {
-    const ideas = getAllIdeas();
-    const idea = ideas.find((idea) => idea.id === ideaId);
+    let idea = findIdeaWithTotalDonationSumWithSameId(ideaId);
 
-    if (!idea) {
-        throw new Error('Error. No idea with such ID found.');
-    }
     return idea.totalDonationSum;
 }
 
 function ideasSumDifference(ideaId) {
-    const ideas = getAllIdeas();
-    const idea = ideas.find((idea) => idea.id === ideaId);
-
-    if (!idea) {
-        throw new Error('Error. No idea with such ID found.');
-    }
+    let idea = checkIfIdeaIdMatchesIdeas(ideaId);
```

Comment:
`check` is not `get`, so it's not clear why this function returns an idea. Let's improve `checkIfIdeaIdMatchesIdeas` function's name

Reply:
Renamed.